js/ui/components/clubhouse.js

```diff
@@ -299,6 +299,10 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
     }
 
     setIcon(actor) {
+        const scaleFactor = St.ThemeContext.get_for_stage(global.stage).scale_factor;
+        actor.width = actor.width * scaleFactor;
+        actor.height = actor.height * scaleFactor;
+
```

Comment:
It doesn't look correct. Usually the scale factor is used to load higher resolution image assets, or translate coordinates correctly. Also, Mutter treats the scale as per monitor, not per stage, which makes this patch technically incorrect (though this is not a problem for us right now since on X11 there's only one "fake" device).

Also, this poses another problem: if this method it called with the same `actor` parameter multiple times, the sizes are scaled incorrectly.

I wonder why do you need to do that for Clubhouse notifications, but not for other Shell notifications.

Since I can't think of a better way to implement this right now, and I'm about to go PTO for the next couple of weeks, I suggest using `actor.set_scale(scaleFactor, scaleFactor);` for now.

Reply:
Using `actor.set_scale(scaleFactor, scaleFactor)` doesn't solve the problem, it does the icon bigger but the positioning is not correct.

The layout for clubhouse notifications is different from the default notifications, I've detected that the icon is not correctly placed because the actor size.

The actor is an `Animation` and it looks like the image is scaled but not the actor size, so the image doesn't fit in the actor space and overflows. Maybe we should add this scale factor to the animation size, to fix this issue.